### PR TITLE
Update safegraph docstrings to pass pydocstyle

### PR DESCRIPTION
**Pull request by chinandrew**
Summary of changes:
- Update docstrings so they pass the pydocstyle linter, which checks against most of PEP257

### CODE REVIEW

File: safegraph/delphi_safegraph/process.py [hunk 1]
```diff
@@ -202,8 +205,9 @@ def process_window(df_list: List[pd.DataFrame],
                    signal_names: List[str],
                    geo_resolutions: List[str],
                    export_dir: str):
-    """Processes a list of input census block group-level data frames as a
-    single data set and exports it.  Assumes each data frame has _only_ one
+    """Process a list of input census block group-level data frames as a single data set and export.
+
+    Assumes each data frame has _only_ one
```
Comment (sgsmob) [inline safegraph/delphi_safegraph/process.py:210]: Combine into single line

**Review by sgsmob — CHANGES_REQUESTED** [review]
Looks good minus the one comment.

File: safegraph/delphi_safegraph/process.py (hunk 1 above)
Comment (chinandrew) [reply safegraph/delphi_safegraph/process.py]: Done

**Review by sgsmob — APPROVED** [review]

**Comment by krivard** [comment]
@chinandrew I think this ran afoul of #368

**Comment by chinandrew** [comment]
> @chinandrew I think this ran afoul of #368

resolved

**Pull request merged**